react-function-components/src/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const container = document.querySelector('#root');
+
+const root = ReactDOM.createRoot(container);
+
+root.render(CustomButton());
```

Comment:
We never directly call a component function, to use a Component we use JSX syntax.
```JSX
root.render(<CustomButton />);
```